index.html

```diff
@@ -7,92 +7,226 @@
-    </div>
-  </div>
+    <section class="green_section">
+      <div class=" section_hero">
+        <div class="section_container flex-j-center flex">
+          <div class="column flex-end-alignment phrase">
+            <div class="catch_phrase">
+              <h1 class="invite_phrase">
+                Work from a&nbsp;cabin near Helsinki
+              </h1>
+              <div class= "button_container flex">
+                <button class="btn" type="submit">Explore</button>
+              </div>
+            </div>
+          </div>
+          <div class="column flex-end-alignment cottage">
+            <div class="photo">
+              <img class="main_photo" src="./switzphoto.jpg" alt="Swiss Cottage">
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
 
 
-  <div class="second_page_container">
-    <div class="paragraph">
-      <p>We offer a handpicked range of cabins for nature lovers in Europeto live and work from for desired time.</p>
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="forest.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Connection</header>
-          <p>We provide a good internet connection in our remotest cabins</p>
+    <section class="section second_page_container">
+      <h2 class="section_heading">
+        We offer a handpicked range of cabins for nature lovers in Europe to live and work
+        from for desired time.
+      </h2>
+      <div class="flex-j-center flex">
+        <div class="pros_container">
+          <div class="photo_description_container">
+            <div class="pros_photo">
+              <img class="pros_pic" src="./forest.png" alt="pic of pros">
+            </div>
+            <div class="header_container flex-j-center flex">
+              <h3 class="paragraph_header">Connection</h3>
+            </div>
+            <div class="paragraph_description flex-j-center flex">
+              <p class="text">
+                We provide a good internet connection in our remotest cabins
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="pros_container">
+          <div class="photo_description_container">
+            <div class="pros_photo high_way">
+              <img class="pros_pic" src="./highway.png" alt="pic of pros">
+            </div>
+            <div class="header_container flex-j-center flex">
+              <h3 class="paragraph_header">
+                Accesibility
+              </h3>
+            </div>
+            <div class="paragraph_description flex-j-center flex">
+              <p class="text">
+                Every Cabin is up in the nature but also easy to get to and to the towns around
+              </p>
+            </div>
+          </div>
+        </div>
```

Comment:
1. `<div class="photo_description_container">` is unnecessary you can remove it
2. Set `max-width` on `.prose_container` to 400px and remove width declaration from it's children
3. To center text inside an element use `text-align: center;` instead of flex
4. Correct text color in description is `#848383`, use color-picker tool 😎 
4. Repeat for each pros_container item in html :)

Reply:
1. fixed
2. fixed
3. this text has been aligned to center by this command... 
4. fixed :) 
5. fixed

Reply:
I think it was using flex centering, but maybe I'm wrong 🤷

Reply:
I am not sure :) nevertheless now it is correct right ? : )

Reply:
@sinkinshadow Nope :) remove flex classes from pros_container children :)

Reply:
hmm If I  remove flexes from childrens I will loose the center of that particular parts of text even if I replace it with text-align...